libs/@local/hql/cst/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@rust/hql-cst",
   "version": "0.1.0",
-  "private": false,
+  "private": true,
```

Comment:
@indietyp can you please make sure that this doesn't happen so frequently? This needs to be private.